app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
Why don't you write a private method to check if @user exists or not ?
If it exists, then go on to 'show' action
Remember use before_action

Reply:
i am not sure how to do it, I will try first.

Reply:
done

Reply:
good

Reply:
This line is not needed anymore because we already have load_user fuction
before loading the show action it will run load_user first, so you will have @user